packages/core/src/App/Containers/Layout/header/default-mobile-links.tsx

```diff
@@ -15,9 +17,16 @@ type TDefaultMobileLinks = {
 const DefaultMobileLinks = React.memo(({ handleClickCashier }: TDefaultMobileLinks) => {
     const { client } = useStore();
     const { has_wallet } = client;
+    const location = useLocation();
+    const walletRoute = location.pathname === routes.wallets;
```

Comment:
you can also utilize the built in web api:
```suggestion
    const walletRoute = window.location.pathname === routes.wallets;
```

Reply:
You don't need to import a third party library for this

Reply:
@aizad-deriv I think it should be fine we are using both approach, I'm using this one and also updated the test case with it

Reply:
![image](https://github.com/binary-com/deriv-app/assets/103104395/5376c55c-f64b-4be1-8b51-57cbb6924f4a)
It is fine I have no issues with it but why are we utilizing third party libraries with a cost whereas we can use the existing web apis for free 🤷🏼‍♂️

Reply:
We can't always go for the "it works" approach we should always go for the sustainable one. Our goal is to make the package lighter so it does not add more runtime to the browser.